ros2service/ros2service/verb/info.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2022 CLOBOT Co., Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ros2cli.node.strategy import add_arguments as add_strategy_node_arguments
+from ros2cli.node.strategy import NodeStrategy
+from ros2service.api import get_service_names_and_types
+from ros2service.api import ServiceNameCompleter
+from ros2topic.verb import VerbExtension
+
+
+class InfoVerb(VerbExtension):
+    """Print information about a service."""
+
+    def add_arguments(self, parser, cli_name):
+        add_strategy_node_arguments(parser)
+        arg = parser.add_argument(
+            'service_name',
+            help="Name of the ROS service to get info (e.g. '/add_two_ints')")
+        arg.completer = ServiceNameCompleter(
+            include_hidden_services_key='include_hidden_services')
+
+    def main(self, *, args):
+        with NodeStrategy(args) as node:
+            service_names_and_types = get_service_names_and_types(
+                node=node,
+                include_hidden_services=args.include_hidden_services)
+            service_name = args.service_name
+
+            for (s_name, s_types) in service_names_and_types:
+                if s_name == service_name:
+                    service_types = s_types
+                    break
+            else:
+                return "Unknown service '%s'" % service_name
+
+            type_str = service_types[0] if len(service_types) == 1 else service_types
+            print('Type: %s' % type_str, end='\n')
```

Comment:
@fujitatomoya
 IMO, a service name has usually one type, and it's better not to be a list format at this time. 
From my perspective, what about adding the following phrase? 





```suggestion
            if len(service_types) > 1:
                        print('There are more than two types in the same service')
            print('Type: %s' % type_str, end='\n')
```
![image](https://user-images.githubusercontent.com/70446214/210173395-791aece9-9c8d-4d05-b674-b6d9bcc129b5.png)

Reply:
It's possible for a service to have multiple types if there are multiple servers advertising with the same name, which is the main debugging situation where I've been using this feature. In that case it's definitely helpful to print all the types to help figure out which server is where.

Reply:
I think these can be added after this PR with `verbose` option aligned with `ros2 topic info -v`, i think that is @leeminju531 's assumption here. btw, which can be https://github.com/ros2/ros2cli/pull/703.